misk/src/main/kotlin/misk/web/exceptions/ExceptionHandlingInterceptor.kt

```diff
@@ -117,6 +105,23 @@ class ExceptionHandlingInterceptor(
     } ?: toInternalServerError(th)
   }
 
+  private fun toGrpcResponse(th: Throwable): GrpcErrorResponse = when (th) {
+    is UnauthenticatedException -> GrpcErrorResponse(GrpcStatus.UNAUTHENTICATED, th.message)
+    is UnauthorizedException -> GrpcErrorResponse(GrpcStatus.PERMISSION_DENIED, th.message)
+    is InvocationTargetException -> toGrpcResponse(th.targetException)
+    is UncheckedExecutionException -> toGrpcResponse(th.cause!!)
```

Comment:
What are these two about?

Reply:
These are copied from `toResponse()` above. AIUI, the idea is to unwrap exceptions that aren't helpful to get the meaningful exception within. For example, I think `InvocationTargetException` gets thrown when wrapping an exception thrown by a Java reflection call.